app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,16 @@
+class PostsController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Turning off CSRF protection is not a good idea. See: https://guides.rubyonrails.org/security.html